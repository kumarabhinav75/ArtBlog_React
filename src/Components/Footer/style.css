footer {
  background: black;
  width: 100vw;
  height: 50px;
  padding: 20px;
  margin-top: 30px;
}
.container-width {
  width: 75%;
  margin: auto;
}

p {
  color: white;
}