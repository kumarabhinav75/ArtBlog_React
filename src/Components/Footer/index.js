import React from 'react';
import './style.css';

const Footer = () => {
  return (
    <footer>
      <div className="container-width">
        <p>&copy; artifact.com 2019</p>
      </div>
    </footer>
  );
}

export default Footer;