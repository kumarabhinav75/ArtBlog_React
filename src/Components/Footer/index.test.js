import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import Footer from './index';

describe('Header', () => {
  it ('should match the snapshot', () => {
    const tree = renderer.create(<Footer />);
    expect(tree).toMatchSnapshot();
  });
})