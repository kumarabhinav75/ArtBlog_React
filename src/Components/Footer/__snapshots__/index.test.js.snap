// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match the snapshot 1`] = `
<footer>
  <div
    className="container-width"
  >
    <p>
      Â© artifact.com 2019
    </p>
  </div>
</footer>
`;
